ansible/user-management/tasks/aws-add-user.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Ensuring AWS IAM User does not exist
+  amazon.aws.iam_user_info:
+    name: "{{ users.aws.username }}"
```

Comment:
this is going to be common for all 5 tasks. Please add an option to give the users as a list and iterate over the same. As is I can only have one user per task unless I am mistaken.

Reply:
Added loop for the same and now multiple users can be created using the list.